src/containers/SearchFilters/SearchFilters.js

```diff
@@ -154,11 +100,11 @@ class SearchFilters extends Component {
                 <FilterIcon className={classes.buttonIcon} />
                 {`${t('filter')} ${total} ${t('results')}`}
```

Comment:
Translations komen wel wat willekeurig voorbij in deze PR. Ben in bovenstaande files 5 of 6 situaties tegengekomen waar translations niet worden gebruikt. Kan deze nog wel even markeren als dat helpt. Anders ga ik ervan uit dat het geen prio heeft voor deze demo.